### DISCUSSION TITLE
Map help

### DISCUSSION THREAD

**Discussion by KDaum50** [Vdauphin/HeartsAndMinds#1391] [Question and answer]
Hello so I've been working for a couple days now, and I got the hearts and minds copied into my mission map everything seems to be working alright with some bugs that I need to get worked out but I'm having an issue with setting up the fraction I tried to follow your example, but the hole code keeps messing with my head any help would be appreciated. I'm trying to achieve the most immersive and realistic game play I can get for my unit. and any tips or trick would help I'll take whatever you got for knowledge to help me out.

**Comment by OverlordZorn** [comment]
whats the exact issue youre facing?

**Reply by KDaum50** [reply]
Right now, it's Changing/Add new faction. I have tried following this http://vdauphin.github.io/HeartsAndMinds/Add-factions I usually do alright when it comes to reading code but there is so much in there it's overwhelming. Do you guys have like a Discord?

**Reply by Vdauphin** [reply]
Hello,
No, we have GitHub discussion
Cheers

**Reply by KDaum50** [reply]
Ok here is what i got so far:
[second array.txt](https://github.com/Vdauphin/HeartsAndMinds/files/9304486/second.array.txt)
[fourth array.txt](https://github.com/Vdauphin/HeartsAndMinds/files/9304489/fourth.array.txt)
now I'm lost on what to do for the Third array this is what i typed up and tried it in game but didnt work at all:
[Third array.txt](https://github.com/Vdauphin/HeartsAndMinds/files/9304497/Third.array.txt)

**Reply by KDaum50** [reply]
I really really want to get this working and get all the features it has to offer setup

**Reply by Vdauphin** [reply]
Why are you following Add-factions wiki page http://vdauphin.github.io/HeartsAndMinds/Add-factions ?
If you want to change the faction, select it in mission parameters and set it to default in param.hpp https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L116-L120

**Reply by KDaum50** [reply]
yes this is what im following not right? http://vdauphin.github.io/HeartsAndMinds/Add-factions

**Reply by Vdauphin** [reply]
Why are you following Add-factions wiki page http://vdauphin.github.io/HeartsAndMinds/Add-factions ?

**Reply by KDaum50** [reply]
Man, I don't know? I thought that was the right thing I'm so confused

**Reply by KDaum50** [reply]
Wait do i just change the numbers?

**Reply by KDaum50** [reply]
Ok I seem to have everything working now thank you guys for the help now on to some small stuff where can I edit IED's and is it possible to disable fake IED's? and how can I add objectives like clearing Minefields or HVT objectives or rescue objective is that type of stuff possible? like I said before my goal is to bring the most immersive deployment that I can get for my Unit.

**Reply by Vdauphin** [reply]
Please create a discussion for each of your question
